app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java

```diff
@@ -53,14 +55,16 @@ public JoinGameImplementation(Activity activity,
                                   ActivityResultLauncher<String[]> requestPermissionsLauncher,
                                   FusedLocationProviderClient fusedLocationClient,
                                   boolean focusable,
-                                  int layoutId){
+                                  int layoutId,
+                                  boolean isLocationMocked){
```

Comment:
There should not be mock location attributes in non-test files.

Reply:
This is the only way I managed to make the tests pass on Cirrus. This parameter can only be set to true via a putExtra.